### ISSUE TITLE
[BUG] localised item name resolver does not resolve custom items

### ISSUE THREAD

**Issue by harbingerofme** [harbingerofme/DebugToolkit#87]
**Describe the bug**
When trying to spawn custom items, pickups, etc. only the numbers and the tokens are accepted.

**Does the bug happen without input**
No

**If input is required, which scenes does it go wrong in**
Ingame

**Comment by SChinchi** [comment]
Necroing this to see if it's worth closing.

From my observations both `give_item` and `create_pick` the token and language name in the autocomplete, which means both should work as an input. Is this something that back in the day used to work differently or am I missing an edge case?

**Comment by SChinchi** [comment]
Yeah, I tried with a mod. I can make some of my own custom items to test specific edge cases if you can think of any. I know we protect against null tokens, but outside of that we just pass the token to `RoR2.Language.GetString` and it does its magic.
